### ISSUE TITLE
Compat tokens: create global compat token module

### ISSUE THREAD

**Issue by tay1orjones** [carbon-design-system/carbon#13690]
Blocked by https://github.com/carbon-design-system/carbon/issues/13686, https://github.com/carbon-design-system/carbon/issues/13688, https://github.com/carbon-design-system/carbon/issues/13689

Once the issues listed above are complete, we should be able to wrap up all the compat token modules into one single module for convienence. This "kitchen sink" or "global" compat token module should be available in both `@carbon/styles` and `@carbon/react` via forwarding all the compat modules from the elements packages listed above.

### Example
Usage for a consumer could be like this:
```scss
@use '@carbon/react/scss/compat/tokens' as *;
```
```[tasklist]
- [ ] Add a new sass module/stylesheet that `@forward`s token modules from the elements packages
- [ ] Ensure this file is included in the build for both `@carbon/react` and `@carbon/styles`
- [ ] Write a sass test to ensure tokens from the 3 elements packages can be used from both `@carbon/react` and `@carbon/styles` (with or without a namespace)
```

**Comment by tay1orjones** [comment]
Since we're changing directions a bit on this, this idea is no longer needed or necessary. v10 tokens will be included in the existing modules rather than in a new module, so a rollup/global module isn't needed.
